content/docs/user-guide/assets/runtime.md

```diff
@@ -0,0 +1,48 @@
+---
+linkTitle: Runtime Asset System
+title: Asset Runtime Asset System
+description: Covers the use and working of the Open 3D Engine (O3DE) runtime asset system.
+weight: 700
+toc: true
+---
+
+## Asset References
+

```

Comment:
I would provide a brief 1-2 line run up to warm folks up. "Assets in O3DE are ..."